src/time.ts

```diff
@@ -37,19 +37,19 @@ export class CronTime {
 	private dayOfWeek: TimeUnitField<'dayOfWeek'> = {};
 
 	constructor(
-		source: CronJobParams['cronTime'],
-		timeZone?: CronJobParams['timeZone'],
+		source: CronJobParams<null>['cronTime'],
```

Comment:
I'm a little confused by these types. wouldn't they be `CronJobParams<null, null>`?

Reply:
the second generic type parameter is optional, so ESLint removes it as superfluous